packages/features/ee/organizations/components/AppInstallButton.tsx

```diff
@@ -0,0 +1,73 @@
+import { PlusIcon } from "lucide-react";
```

Comment:
This is already part of: https://github.com/calcom/cal.com/pull/9337/files, should we remove this file from here?